ktor-utils/jvm/src/io/ktor/util/NioPath.kt

```diff
@@ -22,7 +22,7 @@ public val Path.extension: String get() = fileName.toString().substringAfterLast
 public fun Path.combineSafe(relativePath: Path): File {
     val normalized = relativePath.normalizeAndRelativize()
     if (normalized.startsWith("..")) {
-        throw InvalidPathException(relativePath.toString(), "Bad relative path $relativePath")
+        throw InvalidPathException(relativePath.toString(), "Relative path $relativePath starts with .., which is invalid")
```

Comment:
Maybe, we should rephrase it to `Relative path beginning with .. is invalid` or something like this

Reply:
Will do.